ironfish/src/mining/stratum/clients/client.ts

```diff
@@ -191,11 +197,13 @@ export abstract class StratumClient {
       }
 
       this.logger.info(message)
-    } else {
+    } else if (!this.isClosing) {
       this.logger.info('Disconnected from pool unexpectedly. Reconnecting.')
```

Comment:
should we just move this message into the if statement below so the 2 are tied together?